app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -38,11 +38,16 @@ def form_object_class_map
         private
 
         def setup_form_object(step)
-          "FloodRiskEngine::Steps::#{step.to_s.classify}Form".constantize
-        rescue
-          Rails.logger.debug(" No Form Object found for step [#{step}] - falling back to object_class map")
-          nil
+          form_name = "FloodRiskEngine::Steps::#{step.to_s.classify}Form"
+          begin
+            form_name.constantize
+          rescue => x
```

Comment:
as it's early days  I was just trying to do the dynamic thing without treading on your toes if you wanted to use case  - so I'm happy to tweak this  as everyone sees fit